src/main/java/springsamurais/toyswapbackend/model/Comment.java

```diff
@@ -1,3 +1,78 @@
 package springsamurais.toyswapbackend.model;
 
-public class Comment {}
+import jakarta.persistence.*;
+import java.util.Date;
+
+@Entity
+//@Table(name = "comments")
+public class Comment {
+
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    private Long commentId;
+
+    @Column(nullable = false)
+    private String text;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    //@JoinColumn(name = "", nullable = false)
+    private User commenter;
+
+    @ManyToOne(fetch = FetchType.LAZY)
+    //@JoinColumn(name = "", nullable = false)
+    private Post post;
```

Comment:
Small semantics, but I have referred to user listings as "Listings" in the front-end. If it's not too much work to change all references of "posts" to "listings" that might give more consistency.